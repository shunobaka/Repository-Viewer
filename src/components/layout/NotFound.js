/**
 * @fileoverview Defines a NotFound react component that renders a simple not-found page.
 */
import React from 'react';

/**
 * NotFound react component that renders a simple not-found page.
 */
const NotFound = () => {
  return (
    <section className="landing">
      <div className="landing-inner">
        <h1>The requested page was not found.</h1>
        <h3>Please try again.</h3>
      </div>
    </section>
  );
};

export default NotFound;
