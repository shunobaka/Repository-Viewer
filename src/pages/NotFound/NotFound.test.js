import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

describe('notFound snapshots', () => {
  it('notFound page renders correctly', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
