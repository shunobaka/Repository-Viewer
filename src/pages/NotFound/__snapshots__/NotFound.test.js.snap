// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notFound snapshots notFound page renders correctly 1`] = `
<section
  className="landing"
>
  <div
    className="landing-inner"
  >
    <h1>
      The requested page was not found.
    </h1>
    <h3>
      Please try again.
    </h3>
  </div>
</section>
`;
